firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -18,9 +18,14 @@ struct FieldFocusMessage: Codable {
     let type: String
 }
 
+enum LoginDataMessageName: String {
+    case userSelectedLogin = "RemoteLogins:userSelectedLogin"
+    case allLogins = "RemoteLogins:loginsFound"
```

Comment:
@nbhasin2 @razvanlitianu I added a new message type so that we can handle load vs opt-in differently. This allows us to have the flexibility later if we want to add load behaviour again.